### ISSUE TITLE
BatchNorm: manually set runningMean and runningVar

### ISSUE THREAD

**Issue by minty99** [facebookresearch/flashlight#65]
I am porting PyTorch model to Flashlight. But, I found I can't set runningMean and runningVar of BatchNorm manually; so I modified https://github.com/facebookresearch/flashlight/blob/master/flashlight/nn/modules/BatchNorm.cpp#L48 like this:

```
Variable BatchNorm::forward(const Variable& input) {
  double avg_factor = 0.0;

  if (train_ && trackStats_) {
    ++numBatchesTracked_;
    if (momentum_ < 0) { // cumulative moving average
      avg_factor = 1.0 / numBatchesTracked_;
    } else { // exponential moving average
      avg_factor = momentum_;
    }
  }

  return batchnorm(
      input,
      params_[0],
      params_[1],
      params_[2],
      params_[3],
      featAxes_,
      train_ || (!trackStats_),
      avg_factor,
      epsilon_);
}

void BatchNorm::initialize() {
  Variable wt, bs;
  if (affine_) {
    wt = uniform(featSize_, 0.0, 1.0, af::dtype::f32, true);
    bs = constant(0.0, featSize_, af::dtype::f32, true);
  }
  else {
    wt = Variable();
    bs = Variable();
  }

  if (trackStats_) {
    runningMean_ = constant(0.0, featSize_, af::dtype::f32, false);
    runningVar_ = constant(1.0, featSize_, af::dtype::f32, false);
  }
  params_ = {wt, bs, runningMean_, runningVar_};
}
```

Now, BatchNorm receives 4 parameters; weight, bias, runningMean, runningVar.
How do you think about this?

**Comment by minty99** [comment]
I made PR for this: #70.

**Comment by jacobkahn** [comment]
Closing - we'll continue discussion in the PR.
